importer/helpers/helpers.go

```diff
@@ -83,7 +114,13 @@ func (n *UnixfsNode) AddChild(child *UnixfsNode, db *DagBuilderHelper) error {
 	return nil
 }
 
-func (n *UnixfsNode) setData(data []byte) {
+// Removes the child node at the given index
+func (n *UnixfsNode) RemoveChild(index int) {
+	n.ufmt.RemoveBlockSize(index)
+	n.node.Links = append(n.node.Links[:index], n.node.Links[index+1:]...)
```

Comment:
I think of it as a normal array access, you wouldnt try doing `node.Links[i]` if you werent already sure `i` was in the proper range